catalog/task/push-snapshot/0.10/tests/mocks.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+set -eux
+
+# mocks to be injected into task step scripts
+
+function cosign() {
+  echo Mock cosign called with: $*
+  echo $* >> $(workspaces.data.path)/mock_cosign.txt
```

Comment:
Maybe add the same (and the call count check) for skopeo as well? WDYT?

Reply:
Well, I need to echo out the image string (this is why I have `echo $5`) so I can make sure it does nothing if the source and destination images are the same. So I am not sure how I'd do this cleanly. Any ideas?

Reply:
I meant just add `echo $* >> $(workspaces.data.path)/mock_skopeo.txt` to the top of `skopeo` function. You can still do the `echo $5` afterwards, no issue with that.

Reply:
So the line above may be having some unintended effects in the skopeo call, but it works so I am not going to worry about it. The test was just ensuring it was the same for both images and the echoes are the same anyways so not a big deal